Bottom: 08452cef620327a78c0c1bae781a9e2d9b1a977a
Top:    8faf883fe5de08676d01f7cd7448f3c5a0d673c5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 14:25:15 +0100

Refresh of clean-up-whitespace-in-g-pcb-m

---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index e9469e9..dc594c4 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -483,17 +483,17 @@ PopupMenus =
        {"Unselect all objects" Unselect(All)}
        {"Remove selected objects" RemoveSelected()}
        {"Copy selection to buffer"
-	GetXY(Click to set the snap point for this buffer)
-	PasteBuffer(Clear)
-	PasteBuffer(AddSelected)
-	Mode(PasteBuffer)
+        GetXY(Click to set the snap point for this buffer)
+        PasteBuffer(Clear)
+        PasteBuffer(AddSelected)
+        Mode(PasteBuffer)
        }
        {"Cut selection to buffer"
-	GetXY(Click to set the snap point for this buffer)
-	PasteBuffer(Clear)
-	PasteBuffer(AddSelected)
-	RemoveSelected()
-	Mode(PasteBuffer)
+        GetXY(Click to set the snap point for this buffer)
+        PasteBuffer(Clear)
+        PasteBuffer(AddSelected)
+        RemoveSelected()
+        Mode(PasteBuffer)
        }
        {"Convert selection to element" Select(Convert)}
        {"Auto place selected elements" AutoPlaceSelected()}
diff --git a/src/pcb-menu.res b/src/pcb-menu.res
index aaf48c4..bda9bd7 100644
--- a/src/pcb-menu.res
+++ b/src/pcb-menu.res
@@ -218,7 +218,7 @@ MainMenu =
     {"Set Dispersion" Import(setdisperse)}
    }
   }
-    
+
   {Select
    {"Select all visible objects" Select(All)}
    {"Select all connected objects" Select(Connection)}
@@ -267,7 +267,7 @@ MainMenu =
    {"Elements" ChangeSquare(SelectedElements)}
    {"Pins" ChangeSquare(SelectedPins)}
   }
-    
+
   {Buffer
    {"Copy selection to buffer" GetXY(Click to set the snap point for this buffer)
     PasteBuffer(Clear) PasteBuffer(AddSelected) Mode(PasteBuffer)}
@@ -295,7 +295,7 @@ MainMenu =
    {"#4" checked=buffer,4 PasteBuffer(4) a={"Shift-4" "Shift<Key>4"}}
    {"#5" checked=buffer,5 PasteBuffer(5) a={"Shift-5" "Shift<Key>5"}}
   }
-    
+
   {Connects
    {"Lookup connection to object" GetXY(Click on the object) Connection(Find) a={"Ctrl-F" "Ctrl<Key>f"}}
    {"Reset scanned pads/pins/vias" Connection(ResetPinsViasAndPads) Display(Redraw)}
@@ -331,7 +331,7 @@ MainMenu =
    -
    {"Apply vendor drill mapping" ApplyVendor()}
   }
-    
+
   {Info
    {"Generate object report" ReportObject() a={"Ctrl-R" "Ctrl<Key>r"}}
    {"Generate drill summary" Report(DrillReport)}
